migrating/ossm-migrating-read-me-assembly.adoc

```diff
@@ -0,0 +1,95 @@
+= Important information to know if you are migrating from OpenShift Service Mesh 2.6
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-read-me-assembly
+
+toc::[]
+
+If you are moving from {SMProductName} 2.6 to {SMProductName} 3, read the content in this section first as it contains important information and explanations on the differences between the versions. These differences have a direct impact on your installation and configuration of {SMProduct} 3.
+
+include::modules/ossm-migrating-2-and-3-differences.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-new-operator.adoc[leveloffset=+1]
+
+.Next steps
+You can install the {SMProduct} 3 Operator, or you can run {SMProduct} 2.6 and {SMProduct} 3 in the same cluster using either the multi-tenant deployment model, or the cluster-wide model.
+
+* xref:../install/ossm-installing-openshift-service-mesh.adoc#ossm-installing-operator_ossm-about-deployment-and-update-strategies[Installing the Service Mesh Operator]
+* xref:../install/ossm-running-v2-same-cluster-as-v3-assembly.adoc#ossm-running-v2-v3-multitenant-deployment-model_ossm-running-v2-same-cluster-as-v3-assembly[Running OpenShift Service Mesh 2.6 and OpenShift Service Mesh 3 using multi-tenant model]
+* xref:../install/ossm-running-v2-same-cluster-as-v3-assembly.adoc#ossm-running-v2-v3-cluster-wide-deployment-model_ossm-running-v2-same-cluster-as-v3-assembly[Running OpenShift Service Mesh 2.6 and OpenShift Service Mesh 3 using cluster-wide deployment model]
+
+include::modules/ossm-migrating-read-me-new-resources.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-observability-integrations.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-scoping-discovery-selectors.adoc[leveloffset=+1]
+
+.Next steps
+* xref:../install/ossm-installing-openshift-service-mesh.adoc#ossm-about-discoveryselectors_ossm-scoping-service-mesh-with-discoveryselectors[Scoping Service Mesh with discoverySelectors]
+
+include::modules/ossm-migrating-read-me-sidecar-injection-considerations.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-support-for-multiple-control-planes.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-independently-managed-gateways.adoc[leveloffset=+1]
+
+.Next steps
+If you are using {SMProduct} 2.6, and have not migrated from `ServiceMeshControlPlane` defined gateways to gateway injection, then you must follow the {SMProduct} 2.x gateway migration procedure before you can move to {SMProduct} 3.
+
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-about-gateway-migration_gateway-migration[Gateway migration]
+
+include::modules/ossm-migrating-read-me-explicitly-create-openshift-routes.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-introducing-canary-upgrades.adoc[leveloffset=+1]
+
+.Next steps
+You can set `updateStrategy` to `RevisionBased` to use canary upgrades.
```

Comment:
Update is the preferred term per the [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
You can set `updateStrategy` to `RevisionBased` to use canary updates.
```